bundle/regal/rules/style/prefer_snake_case.rego

```diff
@@ -23,3 +27,13 @@ report contains violation if {
 
 	violation := result.fail(rego.metadata.chain(), result.location(var))
 }
+
+_is_name(ref, pos) if {
+	pos == 0
+	ref.type == "var"
+}
```

Comment:
[golf]
```suggestion
_is_name(ref, 0) if ref.type == "var"
```

Reply:
I like it :) And in this case it won't matter, but I remember how "pattern matching" on function args was noticeably slower when done in the hot path of `walk` traversals.